src/evm/onchain/endpoints.rs

```diff
@@ -389,37 +445,28 @@ impl OnChainConfig {
             }
         );
         println!("fetching abi from {}", endpoint);
```

Comment:
Can you check for a DELEGATECALL contract to see whether the tool can also propagate the ABI of proxy target to proxy during fuzzing on the fly? 

(e.g., we should have PayLuckWinner (0x88b6be2d) in corpus with target as  0x0955A73D014F0693aC7B53CFe77706dAb02b3ef9 in the corpus)